skymap_scanner/utils/prepare_frames.py

```diff
@@ -77,9 +79,7 @@ def prepare_frames(frame_array, baseline_GCD, reco_algo, pulsesName="SplitUnclea
         simclasses,
     )
     
-    nominalPulsesName = "SplitUncleanedInIcePulses"
-    
-    output_frames = []
+    output_frames : list[icetray.I3Frame] = []
```

Comment:
This typehint (subscripting the built-in `list`) is only supported in python 3.11. I suppose that's fine since we're confining the scanner to an image

Reply:
Ah, right. For now I think it's better to stick with the < 3.11 style.

Reply:
For 3.11 follow up in #172